requirements.txt

```diff
@@ -0,0 +1,10 @@
+-e git+https://github.com/SABS-R3-Epidemiology/branchpro.git@219ca92d8c735e3b054a78b7b1a2852ffc97e55e#egg=branchpro
```

Comment:
I don't think you want to include all packages explicitly again. 

```
.
gunicorn==20.0.4
```

Is much better, because now you install branchpro using the setup.py and achieve the same, but also update all dependencies automatically should setup.py change.